README.md

```diff
@@ -1 +1,8 @@
-# talis-node
\ No newline at end of file
+# talis-node
+[![build status](https://travis-ci.org/talis/talis-node.svg?branch=master)](https://travis-ci.org/talis/talis-node)
+
+# Development Version - Not For General Use
+
+This is a development version of the new talis-node version. It is not yet intended for general use.
+This early version pulls existing individual client libraries into one single library with minimal
+changes.
```

Comment:
This is public like the PHP version. When released the PHP version Chris asked us to put a comment to the effect that:

```
This is a development version of the new talis-php version. It is not yet intended for general use. This early version pulls existing individual client libraries into one single library with minimal changes.

Before releasing for general usage there will be major changes to the API. The library will move away from the use of internal Talis project names like Persona, Critic, Babel and Echo etc. Instead it will use more externally relavant names like ListReviews and Files. The API will also move to a more domain driven design rather than the service driven design of the individual libraries

See issue: https://github.com/talis/talis-php/issues/2 and Milestone: https://github.com/talis/talis-php/milestone/1
```

See https://github.com/talis/talis-php.

Should we add a simillar comment, issue and milestone here?